tests/conftest.py

```diff
@@ -96,19 +96,20 @@ def full_input_record():
                 "given_name": "Julio",
                 "family_name": "Cesar",
                 "identifiers": {
-                    "Orcid": "9999-9999-9999-9999"
+                    "Orcid": "0000-0002-1825-0097"
```

Comment:
shelved: we might one to normalize when extracting... or do some sort of logic to allow capitalization, upper and lower case? Applies to more than this line, just got the idea here 😅  If agreed I'll create an issue.

Reply:
Actually, one thought on this: Is/Will be `identifiers` a vocabulary? It can be really messy to have different formats i.e lowercase, uppercase, PascalCase etc... If so, isn't it better to agree on a format and configure via the vocabulary what the users sees?

Reply:
I think all these remarks are covered already in part :grin: . 

- `Orcid` is indeed not visible to the end-user because a small "vocabulary" hides it by displaying a human readable label that corresponds to it. That being said my preference would be for it to be "orcid" instead of "Orcid" for us developers working with it.
- identifier values are normalized according to idutils. (maybe though the remark was just about keys in which case the above point takes care of it)